README.md

```diff
@@ -10,10 +10,15 @@ Download the files in this repository and put them in the same folder.
 To run the tool, install Python 3.9 from [python.org](https://www.python.org) then simply double-click main.py (or run.bat) and the tool will keep trying to switch prime95 to a different core.
 
 **DO NOT FORGET to**:
-open settings.py with a text editor and edit the necessary settings in the ###.
+open settings_thread_switcher_prime95.py with a text editor and edit the necessary settings in the ###.
 
 You have to close and restart the tool for the changes to take effect.
 
+For running aida64, you need to also modify settings_thread_switcher_prime95.py. Note, this bat must be run as admin.
+- To always run as admin, you may:
+1. Create a shortcut to settings_thread_switcher_prime95.py
+2. Right-click -> properties -> shortcut tab -> advanced -> enable "Run as administrator"
+
```

Comment:
Do you mean to refer to the `run_thread_switcher_aida64.bat` file?
Should we put a remark here to change the `python310` command to `py` or `python` if necessary?

Reply:
Done and done